lib/simulation/openfisca/mapping/index.js

```diff
@@ -121,26 +116,18 @@ var applyRessources = function(mesAidesEntity, openFiscaEntity, mappingSchema, s
 
 var setNonInjectedPrestationsToZero = function(openFiscaEntity, dateDeValeur) {
     var periods = getPeriods(dateDeValeur);
-    var concernedPeriods = ['moisPrecedent', 'moisMMoins2', 'moisMMoins3', 'anneeGlissante'];
+    var concernedPeriods = periods.last12Months.concat([periods.rollingYear]);
```

Comment:
I think [it can mean both](http://dictionary.cambridge.org/dictionary/english/concerned), but I'm fine with target, it is clearer.